src/views/BondV2/BondRow.tsx

```diff
@@ -83,24 +84,41 @@ export function BondDataCard({ bond }: { bond: IBondV2 }) {
   );
 }
 
-export function BondTableData({ bond }: { bond: IBondV2 }) {
+export function BondTableData({ bond, networkId }: { bond: IBondV2; networkId: NetworkId }) {
   // Use BondPrice as indicator of loading.
   const isBondLoading = !bond.priceUSD ?? true;
-  // const isBondLoading = useSelector(state => !state.bonding[bond]?.bondPrice ?? true);
+
+  const getEtherscanUrl = (bond: IBondV2) => {
+    if (networkId === NetworkId.TESTNET_RINKEBY) {
+      return `https://rinkeby.etherscan.io/address/${bond.quoteToken}`;
+    }
+    return `https://etherscan.io/address/${bond.quoteToken}`;
+  };
 
   return (
     <TableRow id={`${bond.index}--bond`}>
       <TableCell align="left" className="bond-name-cell">
         <TokenStack tokens={bond.bondIconSvg} />
         <div className="bond-name">
-          <Typography variant="body1">{bond.displayName}</Typography>
-          {bond.isLP && (
-            <Link color="primary" href={bond.lpUrl} target="_blank">
-              <Typography variant="body1">
-                <Trans>Get LP</Trans>
-                <SvgIcon component={ArrowUp} htmlColor="#A3A3A3" />
-              </Typography>
-            </Link>
+          {bond && bond.isLP ? (
```

Comment:
Need to add `<Typography variant="body1">{bond.displayName}</Typography>` for LPs. unless this was a design decision

Reply:
Thanks